src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -108,6 +108,26 @@ CHIP_ERROR ThermostatAttrAccess::Read(const ConcreteReadAttributePath & aPath, A
             return aEncoder.Encode(valueRemoteSensing);
         }
         break;
+    case PresetTypes::Id: {
+        return aEncoder.EncodeList([](const auto & encoder) -> CHIP_ERROR { return CHIP_NO_ERROR; });
+    }
+    break;
+    case Presets::Id: {
+        return aEncoder.EncodeList([](const auto & encoder) -> CHIP_ERROR { return CHIP_NO_ERROR; });
+    }
+    break;
+    case ScheduleTypes::Id: {
+        return aEncoder.EncodeList([](const auto & encoder) -> CHIP_ERROR { return CHIP_NO_ERROR; });
+    }
+    break;
+    case Schedules::Id: {
+        return aEncoder.EncodeList([](const auto & encoder) -> CHIP_ERROR { return CHIP_NO_ERROR; });
+    }
+    break;
+    case QueuedPreset::Id: {
+        chip::app::DataModel::Nullable<chip::app::Clusters::Thermostat::Structs::QueuedPresetStruct::Type> value;
```

Comment:
```suggestion
        DataModel::Nullable<Structs::QueuedPresetStruct::Type> value;
```
This was a comment last time around, but got resolved without fixing....

Reply:
oops must've missed that.  Sorry.